package/rubygem-packaging_rake_tasks.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Fri Apr  1 12:16:56 UTC 2016 - mvidner@suse.com
+
+- Fixed snapper submission (bsc#973607)
+  - check:changelog only before osc:sr
```

Comment:
does the change really do it? for me it looks like, check:syntax and then do "osc:commit" that have its own checks. So actually it looks like it is runned even later. If you want to run it only after osc:commit, then add it after it, not before.